index.html

```diff
@@ -59,7 +59,151 @@ <h1>Hey there. I'm Osugo. <br> <span>I'm a software developer</span></h1>
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work2">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work3">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work4">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work5">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work6">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-me">
+
+        <div class="about-me-text">
+            <h1 class="about-me-topper">About me</h1>
```

Comment:
Kindly I recommend you to use header tag `hx(h2)` tag for the The title (e.g., "My Recent Works") instead of` h1` tag.